VS/VehicleSignals.rdf

```diff
@@ -4365,6 +4372,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;Sonar">
 		<rdfs:subClassOf rdf:resource="&sosa;Sensor"/>
+		<rdfs:label xml:lang="en">Sonar</rdfs:label>
```

Comment:
Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.